packages/dara-components/js/common/table/table.tsx

```diff
@@ -373,12 +373,14 @@ function Table(props: TableProps): JSX.Element {
 
     const datetimeColumns = useMemo(() => getDatetimeColumns(resolvedColumns), [resolvedColumns]);
     const fetchData = useCallback(
-        async (startIndex: number, stopIndex: number) => {
+        async (startIndex: number, stopIndex: number, index?: number) => {
             const response = await getData(combineFilters('AND', [filtersToFilterQuery(filters), searchQuery]), {
+                index,
                 limit: stopIndex - startIndex,
-                offset: startIndex,
+                offset: sortingRules[0] ? startIndex + 5 : startIndex,
```

Comment:
Could you explain what this does and why we're modifying the offset at all?

Reply:
Oops not needed, I was just testing and forgot to return to normal before the commit, will fix